app/config/config.yml

```diff
@@ -7,6 +7,8 @@ imports:
 # http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
     locale: en
+    repositories:
+        - { owner: %repository_username%, name: %repository_name% }
```

Comment:
Both values should be single-quoted strings, as `%` is a reserved Yaml character.